main.go

```diff
@@ -123,3 +145,12 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func getSecret(secretName string) []byte {
+	secret, err := ioutil.ReadFile(dbSecretBasePath + secretName)
```

Comment:
Can we use the filepath library here, e.g. `ioutil.ReadFile(filepath.Join(dbSecretBasePath, secretName))`?